WordPress/Classes/Models/Post+CoreDataProperties.swift

```diff
@@ -4,13 +4,16 @@ import CoreData
 extension Post {
 
     @NSManaged var commentCount: NSNumber?
+    @NSManaged var disabledPublicizeConnections: [NSNumber: [String: String]]?
```

Comment:
@koke yes we can make it non-optinal and the optional handling code does improve a lot. 
I'm happy to make the change if it's ok with everyone @kurzee @jleandroperez 
BTW do you guys have a policy for when an attribute should be non-optional in CoreData? I know by default attributes are crated as optional so maybe that's why most of them are currently this way.